themes/digital.gov/layouts/_default/baseof.html

```diff
@@ -44,8 +44,8 @@
   <div id="clicks_iframe" class="display-none"></div>
 
   <div class="edit_tools">
-    <a class="edit edit-open" aria-label="edit" href="#"><i class="far fa-edit"></i></a>
-    <a class="edit edit-issue" target="_blank" href="https://github.com/GSA/digitalgov.gov/issues/new?body=%0D%0A%2A%2A{{- .Title -}}%2A%2A%0D%0Ahttps%3A%2F%2Fdigital.gov%2F{{- .Permalink -}}%0D%0A%0D%0A---%0D%0A%0D%0A%2A%2APlease+describe+the+issue+clearly.+Include+a+screenshot+if+needed%2A%2A%0D%0A"><i class="fas fa-bug"></i></a>
+    <button class="edit edit-open" aria-label="edit" ><i class="far fa-edit"></i><span class="usa-sr-only">Toggle Edit this page</span></span></button>
```

Comment:
```suggestion
    <button class="edit edit-open" aria-label="edit" type="button"><i class="far fa-edit"></i><span class="usa-sr-only">Toggle Edit this page</span></span></button>
```
Some screenreaders will think it's a submit button if you don't specify otherwise.